test/unit/api_spec.js

```diff
@@ -2275,6 +2277,24 @@ Caron Broadcasting, Inc., an Ohio corporation (“Lessee”).`)
       await loadingTask.destroy();
     });
 
+    it("gets text content with or without includeMarkedContent, and compare (issue 15094)", async function () {
+      const loadingTask = getDocument(buildGetDocumentParams("issue15094.pdf"));
```

Comment:
It seems, as far as I can tell, that this issue is also reproducible using page 568 from the `pdf.pdf` file that we already use in the test-suite.
So how about using that document/page here instead, to avoid adding a new test-file?